src/FSharpPlus/Extensions.fs

```diff
@@ -71,6 +71,16 @@ module Result =
             Ok (f x)
         with e -> Error e
 
+    let getOk (source: Result<'T,'Error>) = match source with Ok x -> x | _ -> invalidArg "source" "Result value was Error"
```

Comment:
I'd go with `get` ... it should probably be discouraged anyway...

Reply:
Ok, let's change it to `get`. Yes, it is as unsafe as `get` from option.